certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+                    {
+                      "name": "custom_header",
+                      "value": "header_value"
+                    }
+                  ],
+                  "httpVerb": "POST",
+                  "url": "https://example.com"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "If the item is not found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Webhooks"
+        ],
+        "summary": "Delete a Webhook",
+        "x-ms-visibility": "internal",
+        "description": "Deletes a Webhook",
+        "operationId": "Webhooks_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Webhook Id",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {          
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "type": "object"
+            },
+            "headers": {}
+          },
+          "204": {
+            "description": "Returns deletion result"
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Webhooks": {
+      "x-ms-notification-content": {
+        "schema": {},
+        "description": "Request From Webhook",
+        "headers": {
+          "\"Request-Context\"": {
+            "description": "\"Request-Context\"",
+            "type": "string"
+          },
+          "\"api-supported-versions\"": {
+            "description": "\"api-supported-versions\"",
+            "type": "string"
+          },
+          "\"x-ms-apihub-cached-response\"": {
+            "description": "\"x-ms-apihub-cached-response\"",
+            "type": "string"
+          },
+          "\"x-ms-apihub-obo\"": {
+            "description": "\"x-ms-apihub-obo\"",
+            "type": "string"
+          },
+          "\"Date\"": {
+            "description": "\"Date\"",
+            "type": "string"
+          },
+          "\"Location\"": {
+            "description": "\"Location\"",
+            "type": "string"
+          },
+          "\"Content-Length\"": {
+            "description": "\"Content-Length\"",
+            "type": "string"
+          },
+          "\"Content-Type\"": {
+            "description": "\"Content-Type\"",
+            "type": "string"
+          }

```

Comment:
@vsorokin-ppmexpress can you please explain what are these headers for?

Reply:
@vsorokin-ppmexpress I left this one unresolved, can you please explain what are these headers for?

Reply:
These headers were generated from Custom Connector UI (from sample), but they are not used within the connector.